streams/src/test/java/org/apache/kafka/streams/processor/internals/StateManagerStub.java

```diff
@@ -34,7 +35,7 @@ public File baseDir() {
 
     @Override
     public void registerStore(final StateStore store,
-                              final StateRestoreCallback stateRestoreCallback) {}
+                              final StateRestoreCallback stateRestoreCallback, final CheckpointCallback checkpoint) {}
```

Comment:
Also in the interest of formatting:
> Use a "single parameter per line" formatting when defining methods (also for methods with only 2 parameters).
https://kafka.apache.org/coding-guide.html